### DISCUSSION TITLE
Users Corrupted

### DISCUSSION THREAD

**Discussion by Niki-29** [Ylianst/MeshCentral#4915] [General]
good evening,
I am writing to you because we have been presenting a security problem with the users since although they have been deleted from the meshcentral UI they continue to appear on the server and we were not able to delete them and they are being used to access our devices in mesh, I appreciate if you could guide us in the process of removing these users.

Thanks for your attention,
I remain attentive to your answer

**Comment by dinger1986** [comment]
So in users on meshcentral these users have been removed?

**Comment by silversword411** [comment]
Huh?
